config.py

```diff
@@ -2556,6 +2598,37 @@ def get_fs_subjects_dir():
                 .resolve())
 
 
+def get_all_contrasts() -> Iterable[ArbitraryContrast]:
+    _validate_contrasts(contrasts)
+    normalized_contrasts = []
+    for contrast in contrasts:
+        if isinstance(contrast, tuple) and len(contrast) == 2:
```

Comment:
By here it **must** by of length 2; unless our validation is broken
```suggestion
        if isinstance(contrast, tuple):
```